controllers/group_totals.go

```diff
@@ -56,48 +54,144 @@ func InstaCountHandler(rw http.ResponseWriter, r *http.Request) {
 		http.Error(rw, err.Error(), http.StatusInternalServerError)
 	}
 
-	queryType := mux.Vars(r)["type"]
-	var pipeline models.Pipeline
-	switch queryType {
-	case "patient":
-		pipeline = models.NewPipeline(group)
-	case "encounter":
-		pipeline = models.NewEncounterPipeline(group)
-	case "condition":
-		pipeline = models.NewConditionPipeline(group)
+	pquery := ""
+	cquery := ""
+	equery := ""
+
+	hasConditionCriteria := false
+	hasEncounterCriteria := false
+
+	for _, characteristic := range group.Characteristic {
+
+		codings := characteristic.Code.Coding
+
+		for _, coding := range codings {
+			//Age
+			if coding.System == "http://loinc.org" && coding.Code == "21612-7" {
+				highAgeDate := time.Date(time.Now().Year()-int(*characteristic.ValueRange.Low.Value), time.Now().Month(), time.Now().Day(), 23, 59, 59, 999, time.UTC).Format("2006-01-02T15:04:05.999")
```

Comment:
In the `time.Date` constructor, the second-to-last argument is _nanoseconds_, not _milliseconds_ -- so 999 means 999 nanoseconds, which isn't what we want.  We want 999000000 nanoseconds, but in Go-land, we can say `999 * time.Millisecond` as a more human-friendly way of saying 999000000.